docusaurus.config.js

```diff
@@ -22,47 +23,31 @@ module.exports = {
             require.resolve('./apify-docs-theme'),
             /** @type {import('./apify-docs-theme/types').ThemeOptions} */
             ({
-                // subNavbar: {
-                //     title: 'Apify Docs',
-                //     items: [
-                //         {
-                //             label: 'Subnav 1',
-                //             href: `${config.absoluteUrl}/platform`,
-                //             target: '_self',
-                //             activeBasePath: 'platform',
-                //             rel: 'dofollow',
-                //         },
-                //         {
-                //             label: 'Subnav 2',
-                //             href: `${config.absoluteUrl}/academy`,
-                //             target: '_self',
-                //             activeBasePath: 'academy',
-                //             rel: 'dofollow',
-                //         },
-                //         {
-                //             label: 'Subnav 3',
-                //             type: 'dropdown',
-                //             items: [
-                //                 {
-                //                     label: 'Reference',
-                //                     href: `${config.absoluteUrl}/api/v2`,
-                //                 },
-                //                 {
-                //                     label: 'Client for JavaScript',
-                //                     href: `${config.absoluteUrl}/client-js/`, // we need a trailing slash here, we'd get redirected there anyway
-                //                     target: '_self',
-                //                     rel: 'dofollow',
-                //                 },
-                //                 {
-                //                     label: 'Client for Python',
-                //                     href: `${config.absoluteUrl}/client-python/`, // we need a trailing slash here, we'd get redirected there anyway
-                //                     target: '_self',
-                //                     rel: 'dofollow',
-                //                 },
-                //             ],
-                //         },
-                //     ],
-                // },
+                subNavbar: {
+                    title: 'Academy',
+                    pathRegex: '/academy',
+                    items: [
+                        {
+                            label: 'Courses',
+                            to: `/academy`,
+                            activeBaseRegex: [
+                                'academy$',
+                                ...collectSlugs(`${__dirname}/sources/academy/webscraping`),
+                                ...collectSlugs(`${__dirname}/sources/academy/platform`),
+                            ].join('|'),
```

Comment:
i dont think we need this, the whole academy is under `academy` url prefix, same for platform docs

i think you can't have a doc inside that would escape this url prefix (defined by the plugin instance)

Reply:
maybe we should change the URL structure first, before we introduce the categories? @mtrunkat 

so it would have common url prefix for each subnavbar section, to get around this craziness. it will work but it feels super ugly

Reply:
huh, so you beat me to it :) i'll still post the explanation I had written, maybe it helps someone

> No yeah, that's exactly why we need that :) This is there to have the "active" class set on the current academy subcategory you are in. The slugs for the documents in the categories are unfortunately semi-random, e.g.:
> - tutorials: "/academy/tutorials", "/academy/node-js/\*", "/academy/python/\*"  etc.
> - courses: "/academy/web-scraping-for-beginners", "/academy/puppeteer-playwright/"...

Reply:
How does this work under the hood? Does it generate some huge client-side list somewhere in the DOM? I'd be a bit afraid of that, but I guess we could live with it as long as it works.

I guess we don't want to have the URL prefixes there, as we've added quite a lot of redirects that are doing the opposite.

Reply:
No, AFAIR this constructs a gigaregex which is used for matching the URLs for the "active" class on the subnavbar... this is resolved during build, the resulting pages just contain the correct class on the correct button for the current page.